src/promptflow/promptflow/_sdk/_load_functions.py

```diff
@@ -63,7 +64,7 @@ def load_common(
             **kwargs,
         )
     except Exception as e:
-        raise Exception(f"Load entity error: {e}") from e
+        raise UserErrorException(f"Load entity error: {e}", privacy_info=[str(e)]) from e
```

Comment:
Think we can create a `LoadFunctionError` that inherits from UserErrorException and use it in all errors inside this py file?

Reply:
I think it might be better to keep the most primitive error type because it is an unknown error and we hope the dashboard will display the specific error type in the end.